src/components/AudioPlayer/AudioPlayer.js

```diff
@@ -111,9 +112,9 @@ class AudioPlayerSub extends Component {
       })
 
       await TrackPlayer.skipToNext()
-      if ((await TrackPlayer.getQueue().length) > 0) {
-        await TrackPlayer.remove(0)
-      }
+      // if ((await TrackPlayer.getQueue().length) > 0) {
```

Comment:
Do we want these removed?

Reply:
Yup, totally missed that. Should be good now